### ISSUE TITLE
Add integration test suite

### ISSUE THREAD

**Issue by parisni** [etesync/server#121]
in order to ease contributions and make the overall more robust would adding tests (eg; docker-compose local stack with postgres  + fastapi api calls on test-client https://fastapi.tiangolo.com/tutorial/testing/) would be of interest ? 

I can work on this

**Comment by tasn** [comment]
Yeah, that would definitely be of interest, though we have multiple test suits. Check out any of the client libraries (etebase-rs for one), they have a full integration test with the server.
